core/services/offchainreporting/contract_tracker.go

```diff
@@ -227,7 +232,7 @@ func (t *OCRContractTracker) HandleLog(lb log.Broadcast) {
 		rr, err = t.contractFilterer.ParseRoundRequested(raw)
 		if err != nil {
 			t.logger.Errorw("could not parse round requested", "err", err)
-			t.logger.ErrorIfCalling(lb.MarkConsumed)
+			t.logger.ErrorIfCalling(func() error { return t.logBroadcaster.MarkConsumed(t.gdb, lb) })
 			return
 		}
 		if IsLaterThan(raw, t.latestRoundRequested.Raw) {
```

Comment:
I feel like this should be transactional with `SaveLatestRoundRequested`.

Unfortunately we cannot be transactional with whatever other DB operations libocr does internally due to the separation of concerns.

Reply:
done